workflow_examples/calc_workflow.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name":"calc_workflow",
+    "num_jobs": 60,
+    "infiles_per_job":1,
```

Comment:
The workflow wouldn't read any input-files, so why would you need to set this parameter?

Reply:
I'm [explicitly checking](https://github.com/HEPCompSim/DCSim/blob/feature/multiple-workflows-per-json/src/SimpleSimulator.cpp#L479-L488), that the keys are given. Therefore,  I would need to provide that information.

I can set it to 0 though, to avoid confusion.

Reply:
Okay, so `0` should be good for now. But you could allso think about type specific requirements. But is not really necessary, I guess.

Reply:
Done: https://github.com/HEPCompSim/DCSim/pull/41/commits/872bf168f39d3886a58a436810d4d335590a9569